Formula/pandoc-citeproc.rb

```diff
@@ -5,8 +5,8 @@ class PandocCiteproc < Formula
 
   desc "Library and executable for using citeproc with pandoc"
   homepage "https://github.com/jgm/pandoc-citeproc"
-  url "https://hackage.haskell.org/package/pandoc-citeproc-0.16.2/pandoc-citeproc-0.16.2.tar.gz"
```

Comment:
It seems you changed the source when updating. Could you change it to be `https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz`?